### ISSUE TITLE
docs not building in readthedocs

### ISSUE THREAD

**Issue by rbtcollins** [testing-cabal/mock#277]
```
 File "/home/docs/checkouts/readthedocs.org/user_builds/mock/envs/latest/local/lib/python2.7/site-packages/sphinx/environment.py", line 147, in __init__
    FileInput.__init__(self, *args, **kwds)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mock/envs/latest/local/lib/python2.7/site-packages/docutils/io.py", line 238, in __init__
    raise InputError(error.errno, error.strerror, source_path)
InputError: [Errno 2] No such file or directory: u'/home/docs/checkouts/readthedocs.org/user_builds/mock/checkouts/latest/docs/changelog.txt'

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/mock/envs/latest/local/lib/python2.7/site-packages/docutils/io.py", line 238, in __init__
    raise InputError(error.errno, error.strerror, source_path)
InputError: [Errno 2] No such file or directory: u'/home/docs/checkouts/readthedocs.org/user_builds/mock/checkouts/latest/docs/changelog.txt'
```

From https://readthedocs.org/builds/mock/3079799/

**Comment by QuLogic** [comment]
`docs/changelog.txt` is a symlink to `../ChangeLog`, but there is no `ChangeLog` anywhere in the tree.

**Comment by rbtcollins** [comment]
ChangeLog is created by pbr when setup.py egg_info is called (directly, or implicitly). But, indeed, that doesn't seem to be getting triggered here. I'll look into that angle - thank you!

**Comment by rbtcollins** [comment]
https://review.openstack.org/202339 should fix this.

**Comment by rbtcollins** [comment]
All fixed - pbr 1.4 and the docs built happily at http://mock.readthedocs.org/en/latest/
